### PR TITLE
refactored async functions for old SDK

### PR DESCRIPTION
**Pull request by zloper**
- First refactoring async functions for old android SDK (coroutines version)

### CODE REVIEW

File: app/src/main/java/lang_import/org/app/ArticleActivity.kt [hunk 1]
```diff
@@ -36,18 +38,18 @@ class ArticleActivity : AppCompatActivity() {
...
-                Log.e("import", "translate", ex)
-                return@whenComplete
-            }
-            val content = "<html><body>${readedTxt}<br/><br/><br/><br/></body></html>"
-            runOnUiThread {
-                webView.settings.javaScriptEnabled = false
-                webView.loadDataWithBaseURL(null, content, "text/html", "UTF-8", null)
+        launch {
+            val res = async { importLang(clearText(intent.extras.getString("discript"))) }.await()
+            fun preparing(res: String) {
```
Comment (reddec) [inline app/src/main/java/lang_import/org/app/ArticleActivity.kt]: Why here a function that just will be invoked immediately after declaration? Can it be inlined?

Comment (zloper) [reply app/src/main/java/lang_import/org/app/ArticleActivity.kt]: yeap, need refactor. This legacy from old variants

**Review by zloper — COMMENTED** [review]
fixed also add first version of new header

**Comment by zloper** [comment]
captureh

**Review by reddec — APPROVED** [review]

**Pull request merged**